src/event.rs

```diff
@@ -54,10 +54,11 @@ pub enum Event {
     NodeAdded(XorName, RoutingTable),
     /// A node has disconnected from us.
     NodeLost(XorName, RoutingTable),
-    /// Our own section has been split, resulting in the included `Prefix` for our new section.
+    /// Our own section has been split, resulting in the included `Prefix`
+    /// for our new section.
```

Comment:
No, but rustfmt has max comment length and it errors if it is exceeded. Not sure what would be the best way to keep the comments at the optimal length. Maybe rustfmt can be configured to do it?

Reply:
Hmm, weird.  It doesn't complain for me at these lines.  I guess you're using rustfmt v0.9.0?

Reply:
The line might have been longer at some point (maybe `Prefix` was `VersionedPrefix` or something), rustfmt complained, I broke it into two lines, then it got shorter later again but there was nothing telling me to reformat it back into single line.

Don't feel like sifting through the code, looking for comments that could be reformatted to take fewer lines to be honest. Unless it can be automated in some way.

Reply:
Absolutely!  I just wanted to check you didn't have some specific editor setting which would cause commits to get polluted with needless changes 😄

Reply:
:+1: